app/src/cli/commands/help.ts

```diff
@@ -0,0 +1,80 @@
+import * as chalk from 'chalk'
+
+import { commands, ICommandModule, IOption } from '../load-commands'
+
+import { $0, dasherizeOption, printTable } from '../util'
+
+const command: ICommandModule = {
+  command: 'help [command]',
+  description: 'HALP!!1',
+  handler({ _: [command] }) {
+    if (command) {
+      printCommandHelp(command, commands[command])
+    } else {
+      printHelp()
+    }
+  },
+}
+export = command
+
+function printHelp() {
+  console.log(chalk.underline('Commands:'))
+  const table: string[][] = []
+  for (const commandName of Object.keys(commands)) {
+    const command = commands[commandName]
+    table.push([chalk.bold(command.command), command.description])
+  }
+  printTable(table)
+  console.log(
+    `\nRun ${chalk.bold(
+      `${$0} help ${chalk.gray('<command>')}`
```

Comment:
This output doesn't look optimized after installing it locally:

<img width="951" src="https://user-images.githubusercontent.com/359239/29700742-91c7ce38-89aa-11e7-93fc-559356f9d022.png">



`Run AppData\Local\GitHubDesktop\app-0.8.1-beta0\GitHubDesktop.exe AppData\Local\GitHubDesktop\app-0.8.1-beta0\resources\app\cli.js help <command> for details about each command`

Reply:
✅ replaced with explicit `github`